core/src/main/java/com/scalar/db/io/Key.java

```diff
@@ -168,6 +172,58 @@ public Key(String columnName, ByteBuffer blobValue) {
     columns = Collections.singletonList(BlobColumn.of(columnName, blobValue));
   }
 
+  /**
+   * Constructs a {@code Key} with a single column with a DATE type
+   *
+   * @param columnName a column name
+   * @param dateValue a DATE value of the column as LocalDate type
+   * @deprecated As of release 3.6.0. Will be removed in release 5.0.0. Use {@link #ofDate(String,
+   *     LocalDate)} instead
+   */
+  @Deprecated
+  public Key(String columnName, LocalDate dateValue) {
+    columns = Collections.singletonList(DateColumn.of(columnName, dateValue));
+  }
+
+  /**
+   * Constructs a {@code Key} with a single column with a TIME type
+   *
+   * @param columnName a column name
+   * @param timeValue a TIME value of the column as LocalTime type
+   * @deprecated As of release 3.6.0. Will be removed in release 5.0.0. Use {@link #ofTime(String,
+   *     LocalTime)} instead
+   */
+  @Deprecated
+  public Key(String columnName, LocalTime timeValue) {
+    columns = Collections.singletonList(TimeColumn.of(columnName, timeValue));
+  }
+
+  /**
+   * Constructs a {@code Key} with a single column with a TIMESTAMP type
+   *
+   * @param columnName a column name
+   * @param timestampValue a TIMESTAMP value of the column as LocalDateTime type
+   * @deprecated As of release 3.6.0. Will be removed in release 5.0.0. Use {@link
+   *     #ofTimestamp(String, LocalDateTime)} instead
+   */
+  @Deprecated
+  public Key(String columnName, LocalDateTime timestampValue) {
+    columns = Collections.singletonList(TimestampColumn.of(columnName, timestampValue));
+  }
+
+  /**
+   * Constructs a {@code Key} with a single column with a TIMESTAMPTZ type
+   *
+   * @param columnName a column name
+   * @param timestampTZValue a TIMESTAMPTZ value of the column as Instant type
+   * @deprecated As of release 3.6.0. Will be removed in release 5.0.0. Use {@link
+   *     #ofTimestampTZ(String, Instant)} instead
+   */
+  @Deprecated
+  public Key(String columnName, Instant timestampTZValue) {
+    columns = Collections.singletonList(TimestampTZColumn.of(columnName, timestampTZValue));
+  }
```

Comment:
We don't. I added them because they were used in a non deprecated Key.of(..) static constructor but did not realize I could use do without creating them.
Thank you. 
It's revised in https://github.com/scalar-labs/scalardb/pull/2437/commits/0b774d21af6fa787e4fd841d42ed03d484d40b99